front/app/components/EventCard/DateBlocks/DayAndMonth.tsx

```diff
@@ -6,13 +6,18 @@ import { EventDay, EventMonth } from './styling';
 // hooks
 import useLocale from 'hooks/useLocale';
 
+// utils
+import { showDotAfterDay } from 'utils/dateUtils';
+import { isNilOrError } from 'utils/helperUtils';
+
 interface Props {
   day?: string;
   month?: string | null;
 }
 
 export default ({ day, month }: Props) => {
   const locale = useLocale();
+  const addDotToDay = !isNilOrError(locale) && showDotAfterDay(locale);
```

Comment:
```suggestion
  const addDotAfterDay = !isNilOrError(locale) && showDotAfterDay(locale);
```

Slightly clearer var.